src/myna/core/__init__.py

```diff
@@ -40,7 +40,7 @@
 
 # Set environment variables on package import
 os.environ["MYNA_INSTALL_PATH"] = os.path.sep.join(
-    os.path.abspath(__file__).split(os.path.sep)[:-3]
+    os.path.abspath(__file__).split(os.path.sep)[:-2]
```

Comment:
Maybe I'm dumb, but why did this change?

Reply:
MYNA_INSTALL_PATH was the base repo directory rather than the package base. "__init__.py" moved from

- `/local/path/Myna/myna/core/__init__.py` -> `MYNA_INSTALL_PATH=/local/path/Myna`

to

- `/local/path/Myna/src/myna/core/__init__.py` -> `MYNA_INSTALL_PATH=/local/path/Myna/src/myna`